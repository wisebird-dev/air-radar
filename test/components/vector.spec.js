import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import Vector from '../../pages/components/vector';

test('Vector renders correctly', async () => {
	const tree = await renderer.create(<Vector/>).toJSON();

	expect(tree).toMatchSnapshot();
});
