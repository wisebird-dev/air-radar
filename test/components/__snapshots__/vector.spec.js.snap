// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Vector renders correctly 1`] = `
.c0 {
  width: 18px;
  height: 18px;
  margin-right: 10px;
  vertical-align: text-top;
}

<img
  className="c0"
/>
`;
